YAXLib/Deserialization.cs

```diff
@@ -450,17 +457,6 @@ private YAXExceptionTypes GetExceptionType(MemberWrapper member)
             : member.TreatErrorsAs;
     }
 
-    private static bool IsAnythingToDeserialize(MemberWrapper member)
-    {
-        if (!member.CanWrite)
-            return false;
-
-        if (member.IsAttributedAsDontSerialize)
```

Comment:
Moved to `IEnumerable<MemberWrapper> GetFieldsToBeSerialized(bool sorted = true)`